packetery/readme.txt

```diff
@@ -71,6 +71,9 @@ We are constantly working on adding new features. You can find a list of feature
 Please contact us at technicka.podpora@zasilkovna.cz .
 
 == Changelog ==
+= 1.0.7 =
+* Fixed: label print page shows user message if no suitable orders are selected
```

Comment:
opravil bych na: the label print page displays a message to the user if no suitable orders are selected